### PR TITLE
Move Licenses and References to after intro

### PR DESCRIPTION
**Pull request by djalova**
I updated the README to match the layout of the other MAX model READMEs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ This experiment creates and trains three neural networks, termed Alice, Bob, and
 
 This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/)
 
+## Model Metadata
+| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
+| ------------- | --------  | -------- | --------- | --------- | -------------- |
+| Cryptography | Adversarial Neural Cryptography | Security | TensorFlow | Auto-generated | N/A |
```
Comment (bdwyer2) [inline README.md]: Maybe you should use the metadata Nick created for this model?
```
| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
| ------------- | --------  | -------- | --------- | --------- | -------------- |
| Security | Cryptography | General | TensorFlow | Randomly generated text | Text |
```

Comment (djalova) [reply README.md]: I'll use this, thanks

**Review by bdwyer2 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ This experiment creates and trains three neural networks, termed Alice, Bob, and
...
+## Licenses
+
+| Component | License | Link  |
+| ------------- | --------  | -------- |
+| This repository | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [LICENSE](LICENSE) |
+| Model Code (3rd party) | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [TensorFlow Models](https://github.com/tensorflow/models/blob/master/LICENSE)|
+
 # Quickstart
 
 ## Prerequisites
```
Comment (MLnick) [inline README.md:28]: While we are making this change I think we should add the detail about "if you don't have an instance of WML" with the IBM Cloud linke

**Review by MLnick — COMMENTED** [review]
LGTM, just left one comment

**Pull request merged**